recipes/anndata/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "anndata" %}
+{% set version = "0.6.4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5d4cc63e46e15b982a36bbad5bacc45fbb8a8fe474210cf991b76f3a5cf2ced0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pandas >=0.21.0
```

Comment:
not sure if pandas/scipy and co is needed here, but pip should be included if you use it above